src/adapters/aave-v2/tests/snapshots/ethereum.profits.json

```diff
@@ -11,93 +11,6 @@
       "chainId": 1,
       "success": true,
       "tokens": [
-        {
-          "address": "0xc713e5e149d5d0715dcd1c156a020976e7e56b88",
-          "name": "Aave interest bearing MKR",
-          "symbol": "aMKR",
-          "decimals": 18,
-          "type": "protocol",
-          "tokens": [
-            {
-              "address": "0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2",
-              "name": "Maker",
-              "symbol": "MKR",
-              "decimals": 18,
-              "profitRaw": "7186772652009n",
```

Comment:
i think we are missing profit formatted number, but should do on separate mr

Reply:
We are not missing them, they are at the bottom.

They get printed like this because we add them at the end of the process (maybe we can apply some order when printing).